.github/workflows/ubuntu_24.yml

```diff
@@ -141,8 +141,8 @@ jobs:
     with:
       runner: 'aks-linux-4-cores-16gb'
       container: '{"image": "${{ fromJSON(needs.docker.outputs.images).ov_test.ubuntu_24_04_x64 }}", "volumes": ["/mount:/mount"]}'
+      affected-components: ${{ needs.smart_ci.outputs.affected_components }}
       python-version: '3.12'
-    if: fromJSON(needs.smart_ci.outputs.affected_components).Python_API.test
```

Comment:
maybe we need to keep PythonAPI or DocsSnippets?

Reply:
I've moved that condition inside the job

Reply:
job has several unconditional steps like dependencies installation, which is always run

Reply:
Yes, and the whole job will be skipped (https://github.com/mryzhov/openvino/blob/gha/smart_ci_snippets/.github/workflows/job_python_api_tests.yml#L32)